makefile

```diff
@@ -0,0 +1,62 @@
+SERVICES = rust-actix
+COMPOSE_FILE_STANDARD = docker-compose.yml
+TEST_COMMAND = pytest tests/test_api_endpoints.py
+LOCAL_DATABASE = ./database/docker-compose.yml
+DATABASE_TABLES = users, followers, messages, latest
+DELAY_TEST_EXECUTION_SECONDS = 3
+
+.PHONY: start-local-db
+start-local-db:
+	@echo "Starting the database container..."
+	docker-compose -f $(LOCAL_DATABASE) up -d
+
+.PHONY: stop-local-db
+stop-local-db:
+	@echo "Stopping and removing the database container..."
+	@docker-compose -f $(LOCAL_DATABASE) stop
+	@docker-compose -f $(LOCAL_DATABASE) rm -f database
+
+.PHONY: clean-db
+clean-db:
+	@echo "Cleaning the database..."
+	@export PGPASSWORD=pass
+	@docker-compose -f $(LOCAL_DATABASE) exec database psql -U user -d waect -c "TRUNCATE TABLE $(DATABASE_TABLES) ;"
+
+.PHONY: start-service
+start-service:
+	@echo "Starting service with Docker Compose file: $(SERVICE)..."
+	docker-compose -f ./$(SERVICE)/$(COMPOSE_FILE_STANDARD) up -d
+
+.PHONY: start-rust-actix
+start-rust-actix:
```

Comment:
Can this be deleted? I don't know if u just used that command to run the rust service?

Reply:
I can delete it sure. I was thinking in a way to run specific tests only with make because then we can put it in the workflow in a way that if we change only e.g. rust service it will execute only the rust files... but it might be possible to set it in the SERVICES variable. Then the workflow will call the test-all and pass only the Services change to the make execution. :) Anyway, I will delete this crap mb 🥹

Reply:
yes please

Reply:
Can we resolve this soon? :))